flocker/provision/_aws.py

```diff
@@ -90,8 +90,9 @@ def _check_if_running(instance):
     Message.new(
         message_type=u"flocker:provision:aws:check_if_running:update",
         instance_state=instance.state,
+        ip_address=instance.ip_address,
     ).write()
```

Comment:
For future reference, you can do `Message.log(message_type=...)` instead of `Message.new(message_type=...).write()`.